test/Spec.hs

```diff
@@ -1,4 +1,9 @@
 import Hedgehog.Main (defaultMain)
```

Comment:
I guess we don't **need** them. We probably won't change this code and break them later. I think we should almost always keep tests around in case they break later especially if they aren't slowing down CI or taking any work to maintain.

Reply:
Fair enough.